Implicit host tags on KVM (#392)
apache/cloudstack-documentation: source/adminguide/host_and_storage_tags.rst

---------
Host tags are responsible for directing VMs to compatible hosts. They are validated with the host tags informed in the compute offerings or in the system offerings.

There are two types of host tags:

> - (Explicit) host tags: the host tags are managed by CloudStack. Users can set, update, and delete host tags via CloudStack API or GUI.

Reviewer: ```suggestion
- (Explicit) host tags: the host tags are managed by CloudStack. Cloud operator can set, update, and delete host tags via CloudStack API or GUI.
```

Author: thanks @andrijapanicsb , committed

> - (Explicit) host tags: the host tags are managed by CloudStack. Cloud operator can set, update, and delete host tags via CloudStack API or GUI.

Author: ```suggestion
- (Explicit) host tags: the host tags are managed by CloudStack, including the flexible host tags. Cloud operator can set, update, and delete host tags via CloudStack API or GUI.
```

> - Implicit host tags: the host tags are not managed by CloudStack API.

Author: ```suggestion
- Implicit host tags: the host tags are not managed by CloudStack API. Cloud operator cannot use both implicit host tags and flexible host tags.
```

.. note::

>    Currently the implicit host tags are only configurable on KVM hosts. 

Author: ```suggestion
   Cloud operator can easily set the implicit host tags per host based on server configurations,
   for example hardware devices (GPU type, disk type, network card type, etc).
   Currently the implicit host tags are only configurable on KVM hosts. 
```
